Parallelise semaphore tests

aliuk2012 opened the pull request:
based on this blog post https://blog.devartis.com/using-parallel-tests-with-semaphore-to-speed-up-rails-builds-838589a93a5

to help setup multiple databases for parallel testing

The reason for the rake test is that semaphore seems to either use its own database.yml file or extends our Database.  Whatever it does it does not create a DB using 'correspondence_platform_test'

```
Created database '4576c3eb-xxx_test'
Created database '4576c3eb-xxx_test2'
Created database '4576c3eb-xxx_test3'
Created database '4576c3eb-xxx_test4'
```

The rake test is called inside a semaphore job. https://semaphoreci.com/aliuk2012/correspondence_tool_staff/settings

aliuk2012 commented on app/views/teams/_business_group_detail.html.slim:
```diff
@@ -3,10 +3,6 @@ h2
     = t('team_leads.director_general')
   = @team.team_lead
 br
```
Could you remove this tag its not needed

aliuk2012 commented on app/views/teams/_directorate_detail.html.slim:
```diff
@@ -3,10 +3,6 @@ h2
     = t('team_leads.director')
   = @team.team_lead
 br
```
Could you remove this tag its not needed

misaka commented:
@beccapearce I agree with @aliuk2012, about the lack of tests. I think that asking you to create a view test from scratch may be a bit much ... unless you really feel like doing it, could I ask you to please edit [CT-1049](https://dsdmoj.atlassian.net/browse/CT-1049) and add an item to the list there to add a spec for the show team view?

aliuk2012 committed:
adds a rake task needed by semaphore

based on this blog post https://blog.devartis.com/using-parallel-tests-with-semaphore-to-speed-up-rails-builds-838589a93a5

to help setup multiple databases for parallel testing.

The reason for the rake test is that semaphore seems to either use its
own database.yml file or extends our Database. Whatever it does it does
not create a DB using 'correspondence_platform_test'

```
Created database '4576c3eb-xxx_test'
Created database '4576c3eb-xxx_test2'
Created database '4576c3eb-xxx_test3'
Created database '4576c3eb-xxx_test4'
```

The rake test is called inside a semaphore job. https://semaphoreci.com/aliuk2012/correspondence_tool_staff/settings
